.github/workflow_scripts/env_setup.sh

```diff
@@ -58,6 +58,8 @@ function install_multimodal {
     # launch different process for each test to make sure memory is released
     python3 -m pip install --upgrade pytest-xdist
     python3 -m pip install --upgrade -e multimodal/[tests]
+    mim install mmcv-full
+    python3 -m pip install --upgrade mmdet
```

Comment:
Just a general comment, no need to take any action in this PR. Shall we extract the testing for object detection from multimodal tests? Just the same as what we do for the old object detection. Open for discussion.

Reply:
Good call out, it would be nice to separate modules to spread out the CI test time. If we continue to add modules like segmentation/ocr into multimodal, its test time would be much much longer than other modules like text and tabular.

Reply:
Do we need to add a unit test for inference?

Reply:
Sure, just added the unit test, thank you.